docs/sequence/error_scenarios/PISPLinkingGrantConsentPhase2.puml

```diff
@@ -0,0 +1,256 @@
+@startuml
```

Comment:
I would make this sequence diagram shorter cutting off everything above `AUTHENTICATION-20`, and give a short textual summary.

Reply:
I prefer it like this. If I was to look at this as a stand alone sequence diagram I wouldn't be able to tell what event initiated the flow that led to the error.

Reply:
Sure - just trying to save you work updating them all ;)